mewarpx/mewarpx/mwxrun.py

```diff
@@ -167,6 +167,21 @@ def _set_grid_params(self):
         else:
             raise ValueError("Unrecognized type of pywarpx.picmi Grid.")
 
+    def get_domain_area(self):
+        """Return float of simulation domain area in X & Y directions or R depending
+        on geometry."""
+        pos_lims = [mwxrun.xmin, mwxrun.xmax,
+                    mwxrun.zmin, mwxrun.zmax]
+
+        if mwxrun.geom_str == "XZ":
+            return (pos_lims[1] - pos_lims[0])
+        elif mwxrun.geom_str == "XYZ":
+            return ((pos_lims[1] - pos_lims[0]) * (pos_lims[3] - pos_lims[2]))
+        else:
+            # TODO: implement RZ when needed
+            warnings.warn('Returning XYZ calculation for domain area')
+            return ((pos_lims[1] - pos_lims[0]) * (pos_lims[3] - pos_lims[2]))
```

Comment:
If I've understood correctly, isn't this simply pi*r^2?

Regardless, I'd suggest raising an 'unimplemented' exception here, rather than warning and returning an incorrect result.